migrations/0029_backfill_interaction_count.sql

```
@@ -0,0 +1,23 @@
+-- Phase 1.7: Backfill interaction_count for existing remote data (Stance A — CRITICAL)
+-- Must run before cleanup worker is enabled. Safe to re-run (full recompute).
+
+UPDATE objects
+SET interaction_count =
+  (SELECT COUNT(*) FROM actor_favourites f
+     JOIN actors fa ON f.actor_id = fa.id
+     WHERE f.object_id = objects.id
+       AND EXISTS (SELECT 1 FROM users u WHERE u.actor_id = fa.id))
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)